plugin/vite.config.template.ts

```diff
@@ -31,7 +31,7 @@ export const web =
   }: {
     name: string;
     parent?: string;
-    type?: "modal" | "plugin" | "core";
+    type?: "modal" | "plugin" | "core" | "popup";
```

Comment:
I think I made a mistake with "plugin", so if you can remove that that would be great!

Reply:
you mean I can build the code without popup type , but what should be the type of popup page

Reply:
![image](https://user-images.githubusercontent.com/89770889/212856630-d1722d6b-0c88-422a-9b63-c4aa52cf5454.png)
![image](https://user-images.githubusercontent.com/89770889/212856715-6ebf944c-403b-49ea-bedd-915ff495377d.png)